projects/VersaGames/config.js

```diff
@@ -0,0 +1,146 @@
+const config = {
+  treasuryAddress: "0x5505ad96d4B1181c00248C8ac2a9A865065FD69D",
```

Comment:
this treasury is a private wallet, not a contract?

Reply:
yes, treasury and operations & funds wallets are private wallets owned by VersaGames

Reply:
fixed, removed private wallets from tvl